### DISCUSSION TITLE
installing extensions

### DISCUSSION THREAD

**Discussion by wibrt** [eclipse-theia/theia#9291] [Q&A]
Hello,

getting to know theia:
how do you get extensions to work?

steps taken:
1. docker run -it -p 3000:3000 theiaide/theia
2. localhost:3000
3. browsing and searching the extensions
4. tried with sqltools
5. this doesnt work? (not able to add a new connection)

am i missing something?

**Comment by vince-fugnitto** [comment]
@wibrt thank you the discussion, were you able to confirm if you noticed any errors in the `dev console` when installing (wondering if its related to the extension itself). Can you confirm if you have issues installing other extensions?

**Comment by wibrt** [comment]
so far i have tried a few extensions: 
not working?:
* PostgreSQL
* SQLTools
* vscode-database

the only one that worked: 
* asciidoc

i toughed it something to the with the host not being found, but a ping from the terminal in theia to the same host works

by _dev console_ you mean _debug console_ ?
(nothing seems to be showing)

**Reply by vince-fugnitto** [reply]
@wibrt I mean the browser's `devtools console`. 
For example, on chrome:
- https://developer.chrome.com/docs/devtools/open/#chrome
